mLRS/Common/hal/esp/rx-hal-bayckrc-900-pa-esp8285.h

```diff
@@ -0,0 +1,117 @@
+//*******************************************************
+// Copyright (c) MLRS project
+// GPL3
+// https://www.gnu.org/licenses/gpl-3.0.de.html
+// OlliW @ www.olliw.eu
+//*******************************************************
+// hal
+//********************************************************
+
+//-------------------------------------------------------
+// BAYCKRC 900 PA RX
+//-------------------------------------------------------
+
+#define DEVICE_HAS_SINGLE_LED
+//#define DEVICE_HAS_NO_DEBUG
```

Comment:
Friendly note - these defines have Debug on the serial.

Reply:
Thanks, I keep forgetting t reset those before committing